File: string-handling/string-position/visual-basic/display_each_character.vb
' Raspberry Pi Foundation
' Developed as part of Ada Computer Science
' Usage licensed under CC BY-NC-SA 4.0
' Note: This file is designed to be copied out and compiled on your machine.
' To run this file you need to:
' 1. Copy the contents
' 2. Paste them into the Visual Basic IDE of your choice (Visual Studio, for example)
' 3. Compile the program
' 4. Run the program

Module Program

    ' The Main subroutine is the default entry point for a VB program
    Sub Main()
        DisplayEachCharacter()
    End Sub

    Sub DisplayEachCharacter()
        Dim password As String = "ItIsAsecReT"

        For index As Integer = 0 To 10
            Console.WriteLine(password(index))
        Next
    End Sub


End Module

File: string-handling/string-position/python/display_each_character.py
# Raspberry Pi Foundation
# Developed as part of Ada Computer Science
# 
# Usage licensed under CC BY-NC-SA 4.0

def display_each_character():
    """Demonstrates how to output each character from a string"""
    password = "ItIsAsecReT"
    
    for index in range(0,11):
        print(password[index])
    

# This code will run if this file is executed directly
# (i.e. not called by another program)
if __name__ == '__main__':
    display_each_character()
